test/test_installation.py

```diff
@@ -113,6 +113,8 @@ def _create_match_list(self) -> List[str]:
             for example_dir in examples_dirs
             for file_path in example_dir.iterdir()
         ]
+        # TODO - why we do not preserve example folders?
```

Comment:
@PatKamin in the repo the examples are inside "examples/basic" folder while after installation all of them are directly in "examples" - is this done intentionally?

Reply:
heh, changed and discussed a little here:

https://github.com/oneapi-src/unified-memory-framework/pull/369#discussion_r1534439916

Reply:
I think we could remove the extra 'basic' directory for simplicity.

Reply:
ok at first I wanted to put my example in a separate directory, but now it is placed in "basic" so I would not change this code